[2024-04-12T20:34:13.293+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T20:34:09.777728+00:00 [queued]>
[2024-04-12T20:34:13.298+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T20:34:09.777728+00:00 [queued]>
[2024-04-12T20:34:13.299+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-04-12T20:34:13.314+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): insert_qdrant> on 2024-04-12 20:34:09.777728+00:00
[2024-04-12T20:34:13.324+0000] {standard_task_runner.py:60} INFO - Started process 2092 to run task
[2024-04-12T20:34:13.346+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '12345678', 'insert_qdrant', 'manual__2024-04-12T20:34:09.777728+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmplo2_xln3']
[2024-04-12T20:34:13.354+0000] {standard_task_runner.py:88} INFO - Job 63: Subtask insert_qdrant
[2024-04-12T20:34:13.429+0000] {task_command.py:423} INFO - Running <TaskInstance: 12345678.insert_qdrant manual__2024-04-12T20:34:09.777728+00:00 [running]> on host 5feba290cccc
[2024-04-12T20:34:13.510+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='12345678' AIRFLOW_CTX_DAG_ID='12345678' AIRFLOW_CTX_TASK_ID='insert_qdrant' AIRFLOW_CTX_EXECUTION_DATE='2024-04-12T20:34:09.777728+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-12T20:34:09.777728+00:00'
[2024-04-12T20:34:13.532+0000] {python.py:202} INFO - Done. Returned value was: {'status': 'success', 'message': 'Data inserted to Vector DB successfully'}
[2024-04-12T20:34:13.568+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=12345678, task_id=insert_qdrant, execution_date=20240412T203409, start_date=20240412T203413, end_date=20240412T203413
[2024-04-12T20:34:13.598+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-12T20:34:13.616+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
